sheetstorm/management/commands/generate_test_data.py

```diff
@@ -33,6 +33,8 @@ class UnsupportedProjectTypeException(Exception):
 class Command(BaseCommand):
     help = "Create initial sample data for SheetStorm application testing."
 
+    types_of_options_dictionary_content = Dict[str, Union[bool, int, None]]
```

Comment:
I think that it would be better to have this variable as OptionsDictType. We need to try follow typing convention.

Reply:
And that's why now I see that `options` variable is not good here and we need to change it to different more meaningful